packages/contentful/src/crn/queries/analytics.queries.ts

```diff
@@ -157,3 +157,72 @@ export const FETCH_TEAM_PRODUCTIVITY = gql`
     }
   }
 `;
+
+export const FETCH_USER_COPRODUCTION = gql`
+  query FetchUserCoproduction($limit: Int, $skip: Int) {
+    usersCollection(order: firstName_ASC, limit: $limit, skip: $skip) {
+      total
+      items {
+        sys {
+          id
+        }
+        firstName
+        lastName
+        nickname
+        alumniSinceDate
+        labsCollection(limit: 3) {
+          items {
+            sys {
+              id
+            }
+          }
+        }
+        teamsCollection(limit: 3) {
+          items {
+            team {
+              sys {
+                id
+              }
+              displayName
+              inactiveSince
+            }
+            role
+            inactiveSinceDate
+          }
+        }
+        linkedFrom {
+          researchOutputsCollection(limit: 20) {
+            items {
+              addedDate
+              sharingStatus
+              authorsCollection(limit: 10) {
```

Comment:
it's possible to add `where: { sys: { id_exists: true }}` so it returns only users not external users
```suggestion
              authorsCollection(limit: 10, where: { sys: { id_exists: true } }) {
```